scrapy/czlscrape/settings.py

```diff
@@ -89,3 +89,7 @@
 #HTTPCACHE_DIR = 'httpcache'
 #HTTPCACHE_IGNORE_HTTP_CODES = []
 #HTTPCACHE_STORAGE = 'scrapy.extensions.httpcache.FilesystemCacheStorage'
+
+LOG_LEVEL = 'WARN'
+LOG_FORMAT = '%(asctime)s srv="czl-scrape" [%(thread)d] %(levelname)s %(name)s: %(message)s'
```

Comment:
* regarding `%(thread)d` vs `%(process)d`, I'll rely on your experience. In Java you'd definitely want the thread since the deployment model is typically done as a single multi-threaded process. As far as I understand, this would be less useful with Python
* if getting the function name is not expensive, then by all means add it to the log message. In Java it would be expensive so we typically avoid it. How about concatenating the module and the function, e.g. `czlscrape.spiders.dialog.parse` ?
* how troublesome is it to produce different `srv` identifiers ? the idea behind it is to be able to easily search through the logs by `srv={x}`